src/video_core/rasterizer_cache.h

```diff
@@ -12,14 +13,26 @@
 
 #include "common/common_types.h"
 #include "core/settings.h"
+#include "video_core/gpu.h"
 #include "video_core/rasterizer_interface.h"
 
 class RasterizerCacheObject {
 public:
+    explicit RasterizerCacheObject(u8* host_ptr)
```

Comment:
This can be made const

```suggestion
    explicit RasterizerCacheObject(const u8* host_ptr)
```

if ToCacheAddr's parameter is made const.